pom.xml

```diff
@@ -6,7 +6,7 @@
 
```

Comment:
Do we not add -preview on line 232, 278 and 314?

Reply:
releaseExt takes care of that